src/status_im/navigation/screens.cljs

```diff
@@ -452,9 +455,24 @@
 
     ;; Settings
 
-    {:name      :settings-password
+    {:name      :screen/settings-password
      :options   options/transparent-modal-screen-options
-     :component settings-password/view}]
+     :component settings-password/view}
+
+    {:name      :screen/change-password
+     :options   (merge
+                 options/transparent-modal-screen-options
+                 {:theme :dark})
+     :component change-password/view}
+
+    {:name      :screen/change-password-loading
+     :options   (merge
```

Comment:
no need to `merge`, assoc is variadic
`(assoc options/.... :theme :dark
                               :popGesture ...
                               `